android/modules/network/src/java/ti/modules/titanium/network/TiHTTPClient.java

```diff
@@ -1009,6 +1009,10 @@ private Object titaniumFileAsPutData(Object value)
 
 	public void send(Object userData) throws UnsupportedEncodingException
 	{
+		//Clear parts and nvPairs before sending new data
+		nvPairs.clear();
+		parts.clear();
+
```

Comment:
It would be better to clear the values only after they have been used:

[L1145](https://github.com/appcelerator/titanium_mobile/pull/9139/files#diff-1e01f142e85df58a3e89a93008e5a21eL1145)

```Java
if (nvPairs.size() > 0) {
	try {
		form = new UrlEncodedFormEntity(nvPairs, "UTF-8");
	} catch (UnsupportedEncodingException e) {
		Log.e(TAG, "Unsupported encoding: ", e);
	}
	nvPairs.clear();
}
```

[L1198](https://github.com/appcelerator/titanium_mobile/pull/9139/files#diff-1e01f142e85df58a3e89a93008e5a21eL1198)
```Java
for(String name : parts.keySet()) {
	Log.d(TAG, "adding part " + name + ", part type: " + parts.get(name).getMimeType() + ", len: "
			+ parts.get(name).getContentLength(), Log.DEBUG_MODE);
	addFilePart(name, parts.get(name));
}
parts.clear();
```